### PR TITLE
PTD-787 Network SVG logos

### PR DESCRIPTION
**Pull request by cryptoiner**
- [x] Removed all SVG logos and replaced with PNG

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@wink-financial/wink-assets",
-  "version": "2.5.8",
+  "version": "2.6.0",
```
Comment (pmprete) [inline package.json:3]: you could use 2.5.9 if it's just an image chain, minor version is used when adding new networks or changing that affect functionality but are not breaking changes.

File: src/attributes/networks.ts [hunk 2]
```diff
@@ -44,22 +44,22 @@ export const NetworksAttributes: { [chainId: number]: INetworkAttributes } = {
   },
   [ChainId.Polygon]: {
     chainId: ChainId.Polygon,
-    logoURL: 'https://polygonscan.com/images/svg/brands/polygon.svg',
+    logoURL: 'https://polygonscan.com/images/svg/brands/polygon.png',
```
Comment (pmprete) [inline src/attributes/networks.ts:47]: I get a 404 error using this url

**Review by pmprete — CHANGES_REQUESTED** [review]
Please change https://polygonscan.com/images/svg/brands/polygon.png url as it gets a 404 error

File: package.json (hunk 1 above)
Comment (cryptoiner) [reply package.json]: Done!

File: src/attributes/networks.ts (hunk 2 above)
Comment (cryptoiner) [reply src/attributes/networks.ts]: Done!

**Review by pmprete — APPROVED** [review]
LGTM

**Pull request merged**